src/SpoolmanManager.cpp

```
@@ -1648,6 +1648,48 @@ int SpoolmanManager::findFilamentNoLock(int vendorId, const char* material,
     return streamFindFilament(vendorId, material, colorHex6, name ? name : "");
 }
 
+void SpoolmanManager::recordLinkResult(int32_t spoolId, bool ok, const char* uid) {
```

Clang (14.0.6)

[warning] parameter name 'ok' is too short, expected at least 3 characters

(readability-identifier-length)